packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -169,6 +176,48 @@ class MessageList extends Widget<Properties> {
     });
   }
 
+  _shouldAddDayHeader(timestamp: undefined | string | number | Date): boolean {
+    const { showDayHeaders } = this.option();
+
+    if (timestamp === undefined || !showDayHeaders) {
```

Comment:
```suggestion
    if (!isDefined(timestamp) || !showDayHeaders) {
```

Reply:
Because dateSerialization and your next condition consider this case.

Reply:
https://github.com/DevExpress/DevExtreme/pull/28160/commits/7207cd514ca4374c8f263c6bc5e49c889e8e4925